bindings/wasm/examples/browser/private_tangle.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {Client, Config, Document, KeyType, Network} from "../../web/identity_wasm.js";
+import {
+    logObjectToScreen,
+    logToScreen,
+} from "./utils.js";
+
+/**
+    This example shows how a DID document can be created on a private tangle.
+    It can be run together with a local hornet node.
+    Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+    for setup instructions.
+**/
+export async function createIdentityPrivateTangle(log = true) {
```

Comment:
Can we add the new examples to the examples test suite (`wasm/bindings/examples/node/test.js` for Nodejs and `wasm/bindings/cypress/integration/browser_test.js`)? I don't expect to start up a private tangle in CI (although that's possible) but we could at least throw a predefined error when `publish_document` fails and test that it is thrown in the tests (so no other e.g. syntax errors creep in).